src/intrinsics/node/utils.js

```diff
@@ -24,7 +24,7 @@ export function getNodeBufferFromTypedArray(realm: Realm, value: ObjectValue): U
 // Takes a value from the host realm and create it into a Prepack Realm.
 // TODO: Move this to a bigger general purpose proxy between the environments.
 // See issue #644 for more details.
-export function createDeepIntrinsic(realm: Realm, value: Value, intrinsicName: string) {
+export function createDeepIntrinsic(realm: Realm, value: any, intrinsicName: string) {
```

Comment:
Why this change? Looks like this might be related to a new Flow version (https://github.com/facebook/prepack/issues/905). If so, I'd prefer a separate pull request that bumps the Flow version number as wel.